apollo-router/src/plugins/response_cache/postgres.rs

```diff
@@ -418,4 +429,246 @@ impl PostgresCacheStorage {
 
         Ok(rec.count.unwrap_or_default() as u64)
     }
+
+    pub(crate) async fn update_cron(&self) -> anyhow::Result<()> {
+        if let Some(cleanup_interval) = &self.cleanup_interval {
+            let cron = Cron::try_from(cleanup_interval)
+                .map_err(PostgresCacheStorageError::WrongCleanupInterval)?;
+            sqlx::query!("SELECT cron.alter_job((SELECT jobid FROM cron.job WHERE jobname = 'delete-old-cache-entries'), $1)", &cron.0)
+                .execute(&self.pg_pool)
+                .await?;
+        }
```

Comment:
Would it be helpful to add some trace-level 'success' logging here for the sake of debugging later?

```suggestion
            log::trace!("Configured `delete-old-cache-entries` cron to have interval = `{}`", &cron.0);
        }
```